scipy/stats/_continuous_distns.py

```diff
@@ -1611,6 +1611,19 @@ def _sf(self, x, a):
                         0.5*sc.gammaincc(a, x),
                         0.5 + 0.5*sc.gammainc(a, -x))
 
+    def _entropy(self, a):
+        a = np.asarray([a])
```

Comment:
This might not be needed. Could you test if tests pass without this line? Other overriden distribution methods do not use it unless they use indexing.

Reply:
No, a bunch of tests fail if that line is removed (44, to be exact).